src/main/java/com/notably/logic/parser/ParserUtil.java

```diff
@@ -10,7 +10,8 @@
  * Contains utility methods used for parsing strings in the various *Parser classes.
  */
 public class ParserUtil {
-
+    private static final String ERROR_PATH = "The input \"%s\" is not valid Path. "
```

Comment:
`not **a** valid path` or `The path \"%s\" is invalid`? Sorry I'm just being a grammar nazi here hehe.

Reply:
oh yes i'll change this